datasets/bg/jud_declarations/bg_jud_declarations.yml

```diff
@@ -0,0 +1,69 @@
+title: Bulgaria Judicial Declarations
+entry_point: crawler.py
+prefix: jud-declarations
```

Comment:
let's make this country-scoped so `bg-`, and then something really short like `jud`